ipa-core/src/protocol/ipa_prf/mod.rs

```diff
@@ -40,6 +40,8 @@ pub(crate) mod aggregation;
 pub mod boolean_ops;
 pub mod prf_eval;
 pub mod prf_sharding;
+pub mod dp_in_mpc;
```

Comment:
Do you think we should keep some separation between DP noise generation for the OPRF (which happens in the clear on each helper) and this DP noise generation for outputs that happens in MPC?   I was originally thinking to create a new `dp_in_mpc` module for this as we have another `dp` module for the OPRF generation.

Reply:
The use of DP mechanisms to create dummy records can be labeled differently, I think.  They can use self-contained DP mechanisms.

FWIW, I think that this is a generic capability, not tied to the ipa_prf module really.

Reply:
I think I follow, so essentially call this DP noise generation DP and call the other one something more specific like OPRF padding.  This one can live outside of ipa_prf while the other one (which is currently outside) would be better in ipa_prf since it particular to this version of ipa. 

Would you be okay putting these code moves and renaming into a later PR?